LICENSE.md

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2015, Vinta Software Studio
+Copyright (c) 2013, Charles Denton and Marc Tamlyn
```

Comment:
I see. Just never saw a license with two copyright lines, not sure that's the right way to do this.   
I think we should have our licence and ALSO provide the one that regards the frontend.   
Maybe have a file with: "related licences" and have only this projects one in the README.
This seems more natural and is how binary distributions do. They have an area with a list of licences.